### PR TITLE
Add "check" alias for testing

### PR DESCRIPTION
**Pull request by strk**
Follows GNU Coding Standards, see
https://www.gnu.org/prep/standards/html_node/Standard-Targets.html#Standard-Targets

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -62,6 +62,9 @@ lint:
 test:
 	for PKG in $(PACKAGES); do go test -cover -coverprofile $$GOPATH/src/$$PKG/coverage.out $$PKG || exit 1; done;
 
+# As for GNU Coding Standard
```
Comment (tboerger) [inline Makefile]: Just skip that comment, nothing anybody cares about :)

**Review by tboerger — CHANGES_REQUESTED** [review]
Please also send it to go-sdk

File: Makefile (hunk 1 above)
Comment (tboerger) [reply Makefile]: And this is also a phony task

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
LGTM

**Comment by andreynering** [comment]
LGTM

**Pull request merged**